src/analytics.ts

```diff
@@ -53,9 +62,7 @@ export function createAnalyticsInstance(options?: Options) {
             growthbookOptions.attributes ??= {}
             growthbookOptions.attributes.id ??= _rudderstack.getAnonymousId()
             growthbookOptions.attributes.country ??=
-                Cookies.get('clients_country') ||
-                (Cookies as any).getJSON('website_status')?.clients_country ||
-                CloudflareCountry
+                Cookies.get('clients_country') || parsedStatus?.clients_country || CloudflareCountry
```

Comment:
nit: Please fix the indentation @aswathy-deriv

Reply:
@ali-hosseini-deriv Actually the printwidth is 120 used and it formats to this on save in the prettier.You want me to change it to 80?

Reply:
Shouldn't they be in the same indentation with other lines?

Reply:
Yes it may change the indentation.If not i can add `// prettier-ignore `and get it indented.or else `"editor.formatOnSave": false` in settings.json.